Eastern-Research-Group/mywaterway
# Feature/540 data page text updates

## Related Issues:
* [HMW-540](https://jira.epa.gov/browse/HMW-540)

## Main Changes:
* Updated the intro text on the Data page.

## Steps To Test:
1. Navigate to http://localhost:3000/data.
2. Confirm that the intro text is correct (check the Jira ticket for reference).
